values.yaml

```diff
@@ -10,6 +10,10 @@ global:
   # opt-in is required, such as by setting `server.enabled` to true.
   enabled: true
 
+  # name sets the prefix used for all resources in the helm chart.
+  # If not set, the prefix will be "${helm release name}-consul".
```

Comment:
Should the notation for this be `{{ .Release.Name }}-consul` or `<helm-release-name>-consul`? Also, if this is not set, you could still set `fullNameOverride`.

Reply:
sure, I'll check what we use throughout the file (if anything).

> Also, if this is not set, you could still set fullNameOverride

True but thought it best not to mention it!

Reply:
It's like our secret menu 😄